test/E2ETest/TyeRunTests.cs

```diff
@@ -767,6 +768,50 @@ public async Task IngressStaticFilesTest()
             });
         }
 
+        [Fact]
+        public async Task IngressAllIPTest()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Does that prompt block the test?

Reply:
It seems that it does. 
I had it when I ran the test locally and it thought that was the reason why the tests maxed out the hour delay in the build job.
It seems to run fine on Linux.